pkgs/test_core/lib/src/runner/configuration/args.dart

```diff
@@ -413,7 +419,9 @@ extension _RuntimeDescription on Runtime {
     var message = StringBuffer('[$identifier]: ');
     message.write('${defaultCompiler.identifier} (default)');
     for (var compiler in supportedCompilers) {
-      if (compiler == defaultCompiler) continue;
+      if (compiler == defaultCompiler) {
+        continue;
+      }
```

Comment:
FWIW I prefer the current single line version.

Reply:
Fixed, I had made this more complicated originally so it didn't fit on one line, but now it does